src/platforms/unreal/configuration/setup-crashreporter.mdx

```diff
@@ -134,6 +132,18 @@ void ConfigureCrashReporter()
 You need to call the `ConfigureCrashReporter` some time after your game
 starts. Any [event attribute](https://develop.sentry.dev/sdk/event-payloads/) can be set.
 
+<Note>
+
+Configuring Crash Reporter attributes should be completed before initializing Sentry SDK if one is enabled. Otherwise, some information could be overwritten and lost.
```

Comment:
```suggestion
You should configure Crash Reporter attributes before initializing the Sentry SDK. Otherwise, some information may get overwritten and lost.
```